site/web/app/themes/sage/assets/styles/components/resource-card.sass

```diff
@@ -0,0 +1,48 @@
+.resource-card
```

Comment:
oopsies, wrong file to work in, please create a file called: _single-resource.sass
in the /site/web/app/themes/sage/assets/styles/layouts directory then link it in the main.scss file.